tests/test_ledger.py

```diff
@@ -232,3 +232,65 @@ def test_delete_non_existent_ledger():
     cashctrl_ledger = CashCtrlLedger()
     with pytest.raises(RequestException):
         cashctrl_ledger.delete_ledger_entry(ids='non-existent')
+
+def test_mirror_ledger(add_vat_code):
+    cashctrl_ledger = CashCtrlLedger()
+    target = StandaloneLedger.standardize_ledger(pd.concat([individual_transaction, collective_transaction]))
+    target['date'] = target['date'].ffill()
+    initial_ledger = cashctrl_ledger.ledger().reset_index(drop=True)
+
+    # Mirror with one single and one collective transaction
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = target.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target accounts were not mirrored'
+        )
+
+    # Mirror with duplicate transactions and delete=False
+    target = StandaloneLedger.standardize_ledger(pd.concat([
+        individual_transaction,
+        individual_transaction,
+        collective_transaction,
+        collective_transaction
+    ]))
+    target['date'] = target['date'].ffill()
+    cashctrl_ledger.mirror_ledger(target=target)
+    mirrored = cashctrl_ledger.ledger().reset_index(drop=True)
+    m = target.merge(mirrored.drop(columns=['id']), how='left', indicator=True)
+    assert len(target) == len(mirrored)
+    assert (m['_merge'] == 'both').all(), (
+            'Mirroring error: Some target ledger were not mirrored'
+        )
```

Comment:
Testing whether individual transaction items match between both sides is not a sufficient test. In theory, there could be a mismatch between transaction item and ids. A better test is to convert transactions to unique string representations and ensure the number of occurrences is identical on both sides.

Maybe it would be helpful to write a utility function that converts a ledger data frame into unique string representations. This function could be used both in the package module and in tests.